pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/con_pantheon_yaml_file.adoc

```diff
@@ -17,17 +16,48 @@ The configuration file enables you to import content from a Git repository into
 The {ProductShortName} configuration file defines the following parameters:
 
 * `server` The URL of the {ProductShortName} server.
-* `repositories` A multi-valued structure with the following child members:
+* `repository` The name of the repository. It is recommended not to include spaces in the repository name.
+* `variants` Separate sets of output built on the same set of files and differentiated by these values:`name, `path`, and `canonical`. Modules and assemblies can have variants to differentiate how they are used, and what output is created.
 +
-** `name` (optional) A unique ID that displays in URLs on the {ProductShortName} site when you view modules and assemblies.
+** `name` (optional) A unique ID that displays in URLs on the {ProductShortName} site when you view modules and assemblies. If no value is provided for `name`, this field is defined as *DEFAULT*.
 +
 For example, if you specify `name: rhel_docs`, and one of your files is named [filename]`foo.adoc`, the module's URL in {ProductShortName} is:
-`\http://<_{ProductShortName}__url>/content/repositories/**rhel_docs**/entities/modules/getting_started/foo.adoc.preview`
-** `modules` (optional) Patterns for modules that show up in the search results on {ProductShortName}.
+`\https://<_{ProductShortName}__url>/content/repositories/*repository*/entities/modules/getting_started/foo.adoc.preview`
+** `path` The path containing the resource files (for example attributes.adoc).
+** `canonical` The primary or original variant of content. If a search engine sees multiple variants of the same piece of content (HTML page or a group of pages) it needs to know what the primary or original one is to establish ranking. Declaring a piece of content as canonical tells the search engine to treat that piece of content as the original or the `canonical` variant. Declaring canonical to be true or false is translated into HTML meta data. Possible values: *true* or *false*.
+* Examples of variants:
+** Document level - _Installing RHV using the command line_; _Installing RHV using the Administration Portal_.
+** Module level - The same module is published in multiple Red Hat documents, such as _Red Hat Decision Manager_, _Red Hat Process Automation Manager_.
+
+.Example YAML file with multiple variants
```

Comment:
The example provided here does not actually contain multiple variants. It contains only one variant `rhel9`. An example of multiple variants could be:
```
variants:
  - name: rhel9
    path: rhel-9/common-content/attributes.adoc
    canonical: true
  - name: rhel8
    path: rhel-8/common-content/attributes.adoc
    canonical: false
```